pkgs/applications/blockchains/framesh/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, fetchurl, appimageTools, ... }:
+
+let
+  pname = "framesh";
+  version = "0.5.0-beta.19";
+  name = "${pname}-${version}";
+  src = fetchurl {
+    url = "https://github.com/floating/frame/releases/download/v${version}/Frame-${version}.AppImage";
+    sha256 = "sha256-/mmVnUnJVfoKfXMN2zPlo+lsUninCNuENmVA2x8dDEY=";
+  };
+
+  appimageContents = appimageTools.extractType2 {
+    inherit name src;
+  };
+in
+appimageTools.wrapType2 {
+  inherit name src;
+
+  extraInstallCommands = ''
+    ln -s $out/bin/${name} $out/bin/${pname}
+    install -m 444 -D ${appimageContents}/frame.desktop $out/share/applications/frame.desktop
+    install -m 444 -D ${appimageContents}/frame.png \
+    $out/share/icons/hicolor/512x512/apps/frame.png
```

Comment:
Minor nitpick: maybe indent those a bit more so it's easier to read? Something like

```bash
install -m 444 -D ${appimageContents}/frame.png \
    $out/share/icons/hicolor/512x512/apps/frame.png
```

Reply:
added indentation for the multiline command